src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -46,6 +46,9 @@ public AddCommand(String name,
         for (String tagName : tags) {
             tagSet.add(new Tag(tagName));
         }
+
+        name = toCorrectCase(name);
```

Comment:
It seems to me that it might be better to uniformize the case in the comparator of person.